src/main/java/appeng/parts/misc/ToggleBusPart.java

```diff
@@ -81,6 +81,7 @@ public ToggleBusPart(IPartItem<?> partItem) {
 
         this.getMainNode().setIdlePowerUsage(0.0);
         this.getMainNode().setFlags(GridFlags.PREFERRED);
+        this.getMainNode().setFlags(GridFlags.REQUIRE_CHANNEL);
```

Comment:
this is wrong, setFlags completely overrides the flags I think. please go in creative mode and double check everything :P